src/Mod/Draft/draftguitools/gui_planeproxy.py

```diff
@@ -43,8 +43,8 @@ class Draft_WorkingPlaneProxy:
     def GetResources(self):
         """Set icon, menu and tooltip."""
         return {"Pixmap": "Draft_PlaneProxy",
-                "MenuText": QT_TRANSLATE_NOOP("Draft_WorkingPlaneProxy", "Create working plane proxy"),
-                "ToolTip": QT_TRANSLATE_NOOP("Draft_WorkingPlaneProxy", "Creates a proxy object from the current working plane.\nOnce the object is created double click it in the tree view to restore the camera position and objects' visibilities.\nThen you can use it to save a different camera position and objects' states any time you need.")}
+                "MenuText": QT_TRANSLATE_NOOP("Draft_WorkingPlaneProxy", "Working Plane Proxy"),
+                "ToolTip": QT_TRANSLATE_NOOP("Draft_WorkingPlaneProxy", "Creates a proxy object from the current working plane and allows to (re)store the camera position and visibility of objects")}
```

Comment:
```suggestion
                "ToolTip": QT_TRANSLATE_NOOP("Draft_WorkingPlaneProxy", "Creates a proxy object from the current working plane and allows to restore the camera position and visibility of objects")}
```
I don't see why the brackets were needed